cron

```diff
@@ -1,7 +1,8 @@
 MAILTO=fanglin@uw.edu
```

Comment:
Could this be converted to a mailman list?

Reply:
for now just me